### ISSUE TITLE
Mark "duplicate-id" rule as a best practice rule 

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-web#4102]
Based on feedback from users we tested to check the user impact of duplicate ID failures on static elements. We found no user impact on the experience with any of the ATs including Narrator, JAWS, NVDA, Dragon, Windows Speech Recognition.

After further discussions with the Tooling Advisory Board, we decided to make this a best practice rule

## Context
- AI-Web Version : Prod 2.24.0

**Comment by tohmjudson-ms** [comment]
Happy to pick this fix up

**Comment by ferBonnin** [comment]
thanks @tohmjudson-ms! assigning to you and marking this as active

**Comment by dbjorge** [comment]
Resolved by @tohmjudson-ms in #4127

**Comment by ferBonnin** [comment]
this is now in production, @iamrafan can you please verify and close?

**Comment by iamrafan** [comment]
Validated in AI Web Prod 2.26.0
